sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -3130,6 +3130,23 @@ public void testUnnestArrayColumn() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
+  @Test
+  public void testStringAggregation() {
+    String sql =
+        "SELECT STRING_AGG(fruit) AS string_agg"
+            + " FROM UNNEST([\"apple\", \"pear\", NULL, \"banana\", \"pear\"]) AS fruit";
```

Comment:
In fact, I think it was a decision to not handle NULL in string coder. 

So the easiest way to move forward is to not include `NULL` in your test case, and leave a comment to say that `STRING_AGG` does not handle `NULL` value